packages/client/hmi-client/src/components/documents/Document.vue

```diff
@@ -1,253 +1,266 @@
 <template>
-	<section class="asset" v-if="doc" ref="sectionElem">
-		<div class="two-columns">
-			<div class="content-navigator" v-if="isEditable">
-				<!-- TODO: Set these buttons up to toggle between PDF and extraction view -->
-				<div class="p-buttonset content-switcher">
-					<Button
-						class="p-button-secondary p-button-sm"
-						label="Extractions"
-						icon="pi pi-list"
-						outlined
-						:active="true"
-					/>
-					<Button
-						class="p-button-secondary p-button-sm"
-						label="PDF"
-						icon="pi pi-file"
-						outlined
-						:active="false"
-						disabled
-					/>
-				</div>
-
-				<!-- TODO: Add scroll to anchor feature -->
-				<div class="scroll-to-section-links">
-					<a>Top</a>
-					<a v-if="!isEmpty(formattedAbstract)">Abstract</a>
-					<a v-if="doc?.knownEntities?.summaries?.sections">Section summaries</a>
-					<a v-if="!isEmpty(figureArtifacts)"
-						>Figures <span class="artifact-amount">({{ figureArtifacts.length }})</span></a
-					>
-					<a v-if="!isEmpty(tableArtifacts)"
-						>Tables <span class="artifact-amount">({{ tableArtifacts.length }})</span></a
-					>
-					<a v-if="!isEmpty(equationArtifacts)"
-						>Equations <span class="artifact-amount">({{ equationArtifacts.length }})</span></a
-					>
-					<a v-if="!isEmpty(githubUrls)"
-						>GitHub URLs <span class="artifact-amount">({{ githubUrls.length }})</span></a
-					>
-					<a v-if="!isEmpty(urlArtifacts)"
-						>Other URLs <span class="artifact-amount">({{ urlArtifacts.length }})</span></a
-					>
-					<a v-if="!isEmpty(otherArtifacts)"
-						>Other extractions <span class="artifact-amount">({{ otherArtifacts.length }})</span></a
-					>
-					<a v-if="!isEmpty(doc.citationList)"
-						>References <span class="artifact-amount">({{ doc.citationList.length }})</span></a
-					>
-					<a v-if="!isEmpty(relatedTerariumArtifacts)"
-						>Associated resources
-						<span class="artifact-amount">({{ relatedTerariumArtifacts.length }})</span></a
-					>
-				</div>
-
-				<!-- TODO: Add search on page function (highlight matches and scroll to the next one?)-->
-				<!--- 
+	<section class="two-columns asset" v-if="doc" ref="sectionElem">
+		<nav v-if="isEditable">
```

Comment:
The tables of content should be responsive and not based if it's editable or not.
Maybe a dropdown next to the buttonset. talk to Jamie about it.